app/package.json

```diff
@@ -38,6 +38,7 @@
     "@xmpp/client": "^0.13.1",
     "axios": "^1.6.2",
     "dompurify": "^3.1.1",
+    "email-addresses": "^5.0.0",
```

Comment:
Good choice of library: No dependencies, small code, and focus on RFC compliance.
I'll try to use that in other places where we need as well.